B827EBFFFEEB4420.json

```
@@ -0,0 +1,19 @@
+
+{
+  "gateway_conf": {
+    "gateway_ID": "eui-b827ebfffeeb442",
```

```suggestion
    "gateway_ID": "B827EBFFFEEB4420",
```

please use capitals and no prefix here.